package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "node-red-desktop",
+  "version": "0.0.1",
+  "main": "main.js",
+  "build": {
+    "appId": "com.electron.node-red",
+    "productName": "Node-RED",
+    "files": [
+      "**/*",
+      {
+        "from": "node_modules/@node-red",
+        "to": "node_modules/@node-red"
+      }
+    ],
+    "win": {
+      "target": [
+        {
+          "target": "msi"
+        }
+      ]
+    },
+    "mac": {
+      "category": "public.app-category.developer-tools",
+      "target": [
+        {
+          "target": "dmg"
+        }
+      ]
+    },
+    "linux": {
+      "category": "Development",
+      "target": [
+        {
+          "target": "deb",
+          "arch": [
+            "x64",
+            "armv7l"
+          ]
+        },
+        {
+          "target": "rpm"
+        }
+      ],
+      "publish": []
+    }
+  },
+  "scripts": {
+    "start": "electron .",
+    "build": "electron-builder"
+  },
+  "dependencies": {
+    "node-red": "1.1.3",
+    "node-red-dashboard": "^2.23.3"
```

Comment:
Should we be including dashboard by default?

Reply:
Good point - no - not if this is focussed around the editor.

Reply:
I agree with that. I removed the dashboard from the dependency modules.